content/concepts/target-market-settings.md

```diff
@@ -171,9 +187,64 @@ Below is the JSON standard format representing a measure family.
 }
 ```
 
+::: panel-link Want more details about the measure family resource? [Check its endpoints here!](/api-reference.html#Measurefamily)
+:::
+
+## Measurement family
+::: availability versions=Serenity editions=CE,EE
+:::
+
+If you want to store measurement regarding your product such as weight, height or power inside your PIM, you will need measurement families. These entities will be really helpful in the case you are requesting products for a given channel and you want these measurement attributes to be converted into the units you specified in your channel.
+
+Below is an example of one of these measurement attribute.
+
+![Metrics attribute](/img/concepts/metrics_attributes_ui.png)
+
+Below is the JSON standard format representing a measurement family.
+
+```json
+{
+  "code": "AREA",
+  "labels": {
+    "en_US": "Area",
+    "fr_FR": "Surface"
+  },
+  "standard_unit_code": "SQUARE_METER",
```

Comment:
I would add in the `units` array below this unit to be extra clear.

Reply:
Ah yes, it's added